Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -2329,8 +2329,17 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
                   // Check that the node hasn't got any parents from manual curation
                   !topicGraphNode.isManuallyCurated
             else { return }
-            topicGraph.addEdge(from: topicGraphParentNode, to: topicGraphNode)
-            automaticallyCuratedSymbols.append((child: reference, parent: parentReference))
+            if let overloadGroup = linkResolver.localResolver.overloadGroup(of: topicGraphNode.reference),
+               let overloadGroupNode = topicGraph.nodeWithReference(overloadGroup)
+            {
+                // If this symbol is part of an overload group, curate it under the overload group
+                // instead of its original parent
+                topicGraph.addEdge(from: overloadGroupNode, to: topicGraphNode)
+                automaticallyCuratedSymbols.append((child: reference, parent: overloadGroup))
+            } else {
+                topicGraph.addEdge(from: topicGraphParentNode, to: topicGraphNode)
+                automaticallyCuratedSymbols.append((child: reference, parent: parentReference))
+            }
```

Comment:
The topic graph typically doesn't skip edges. For example, when a topic is manually curated it still has the edge for the automatic curation. Unless we generally want to change what edges/relationships the topic graph stores, I believe that the overloaded symbols should still have their member-of edges. 

For the edges from the overloaded symbols to the overload group; I wonder if `SymbolGraphRelationshipsBuilder` would be a better fit for that, since that's responsible for adding special purpose edges for other types of relationships.

Reply:
I'd have to look into what `SymbolGraphRelationshipsBuilder` does; i'm not familiar with it. From what i remember, isn't it the one responsible for creating override/default-implementation/etc relationships? Are you recommending that i add a new RenderNode section with that?

Reply:
No, from what I understand we don't need a new section for this, only to add the edges between the overload group and the grouped overloaded symbols in the topic graph so that it can be retrieves later.